jobsub/examples/gbl_local/config.cfg

```diff
@@ -51,10 +51,10 @@ HotpixelRunNumber	= @RunNumber@
 SkipNEvents		= 0
 
 # Output subfolder structure
-DatabasePath		= ./output/database
-HistogramPath		= ./output/histograms
-LcioPath            = ./output/lcio
-LogPath				= ./output/logs
+DatabasePath		= %(BasePath)s/output/database
```

Comment:
Oooh, now I see it. I'm using the `--subdir` option, hence he's looking for the output directory from there.

I like the `--subdir` option, because it keeps your workspace clean. But if you prefer the basic setup without this option, I can adapt the commit accordingly. Any opinion?